static/templates/admin_user_list.handlebars

```diff
@@ -6,45 +6,47 @@
   <td>
     <span class="email">{{email}}</span>
   </td>
-  {{#if is_bot}}
+  {{#if is_user_admin}}
```

Comment:
Yeah, renamed it to `is_loggedin_user_admin` that seems more appropriate. As current user can refer to the user in iteration.